src/main/resources/de/thetaphi/forbiddenapis/gradle/plugin-init.groovy

```diff
@@ -23,10 +23,19 @@ if (project.plugins.withType(JavaBasePlugin.class).isEmpty()) {
   throw new PluginInstantiationException('Forbidden-apis only works in projects using the java plugin.');
 }
 
+// chck if running in Gradle Daemon?
+// see: http://stackoverflow.com/questions/23265217/how-to-know-whether-you-are-running-inside-a-gradle-daemon
+boolean isGradleDaemon = System.getProperty('sun.java.command', '').startsWith('org.gradle.launcher.daemon.') ||
```

Comment:
The second part with stack trace should hit on any JVM. The first check is just a shortcut.